jest/global.js

```diff
@@ -2,13 +2,18 @@ import $ from 'jquery';
 
 global.$ = global.jQuery = $;
 
+const userFn = {
+  anonymousId: jest.fn(),
+};
+
 global.analytics = {
   page: jest.fn(),
   track: jest.fn(),
   identify: jest.fn(),
+  user: jest.fn(() => userFn),
 };
 
-global._DATADOG_SYNTHETICS_BROWSER = false;
+global.global._DATADOG_SYNTHETICS_BROWSER = false;
```

Comment:
why do we have two: `global.global` ?

Reply:
was a typo! Good catch 😊  fixed in https://github.com/circleci/circleci-docs/pull/6427/commits/476e19cf6d1f4b79bc422dfd9cef48c04ec00230